sdk/compiler/damlc/tests/src/DamlcTest.hs

```diff
@@ -33,7 +33,7 @@ main = withSdkVersions $ do
 
     -- TODO https://github.com/digital-asset/daml/issues/12051
     --   Remove once Daml-LF 1.15 is the default compiler output
-    script1DevDar <- locateRunfiles (mainWorkspace </> "daml-script" </> "daml" </> "daml-script-1.dev.dar")
+    script1DevDar <- locateRunfiles (mainWorkspace </> "daml-script" </> "daml" </> "daml-script-1.17.dar")
```

Comment:
Another example
Also the TODO above has a closed ticket, perhaps we can remove it